api/auth.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const passport = require('../auth/index');
+const router = require('express').Router();
+
+router.get('/auth/facebook', passport.authenticate('facebook', { scope: 'email' }));
+router.get('/auth/facebook/callback', passport.authenticate('facebook',
+  { successRedirect: '/api/users', failureRedirect: '/login' }));
```

Comment:
And so I did take care of it in this particular commit, since it happened when I started working on frontend it created a little confusion here, so you can check it out here, or later when you check out later commits
[--->](https://github.com/DorianSyruss/wall-of-sayings/pull/1/commits/eeef20751b50776e686c98b725b3475791da42e9)